build-support/bin/go/mac/10.10/1.10.5

```diff
@@ -0,0 +1 @@
+../10.13/1.10.5
```

Comment:
I see in your commit message that this is a symbolic link. Is it a relative path specifically? Would be helpful to add a brief comment
```suggestion
# Relative link
../10.13/1.10.5
```

Reply:
This is how git diffs render symbolic links.

Reply:
Ah that makes much more sense. Looks good then!